Demo/Demo/DemoSettings.swift

```diff
@@ -0,0 +1,21 @@
+// swiftlint:disable convenience_type
+
+import Foundation
+
+class DemoSettings {
+
+    enum Environment {
+        case sandbox, production
+
+        var baseURL: String {
+            switch self {
+            case .sandbox:
+                return "https://ppcp-sample-merchant-sand.herokuapp.com"
+            case .production:
+                return "https://ppcp-sample-merchant-prod.herokuapp.com"
```

Comment:
note for later - maybe we update the heroku app to remove `ppcp` from the URL string, if we're going to take over that merchant app?

Reply:
Good call. Yeh, we have a few things to do with that server, including to move it to the PayPal github. We'll add this rename to this list too!